### DISCUSSION TITLE
Folder that redirects to another repository

### DISCUSSION THREAD

**Discussion by bernaljuanse** [community/community#23485] [New to GitHub]
<p>I have a question, how do I make it so that when I click on a folder, it redirects me to another repository, in this example the folder “core @ aacbc90” when I click it redirects to another repository other than the one in the folder.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/7/47252b10ea2bf575706beaea947f5cf38c10d9f5.png" data-download-href="/uploads/short-url/a9nx2Hry3ssFfudvMEjsvIrKSUd.png?dl=1" title="imagen" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/32989140/181077355-bd58ab22-f153-4cec-a311-92a7ba1027ea.png" alt="imagen" data-base62-sha1="a9nx2Hry3ssFfudvMEjsvIrKSUd" width="690" height="148" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/7/47252b10ea2bf575706beaea947f5cf38c10d9f5_2_690x148.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/7/47252b10ea2bf575706beaea947f5cf38c10d9f5_2_1035x222.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/7/47252b10ea2bf575706beaea947f5cf38c10d9f5_2_1380x296.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/7/47252b10ea2bf575706beaea947f5cf38c10d9f5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">imagen</span><span class="informations">1409×303 26.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
here is an example of what I want to achieve, there I am at the url <a href="https://github.com/adaptlearning/adapt_framework/tree/master/src" class="inline-onebox" rel="noopener nofollow ugc">adapt_framework/src at master · adaptlearning/adapt_framework · GitHub</a>, but when I click on “core @ aacbc90” it redirects me to https:// <a href="http://github.com/adaptlearning/adapt-contrib-core/tree/aacbc90cc7eeedaea649270303a695c9fc7f45a0" class="inline-onebox" rel="noopener nofollow ugc">GitHub - adaptlearning/adapt-contrib-core at aacbc90cc7eeedaea649270303a695c9fc7f45a0</a>. Thanks for the help.

**Comment by airtower-luna** [comment, marked as answer]
<p>That’s a <em>submodule</em>, basically a reference to another repository at a specific commit. See: <a href="https://git-scm.com/book/en/v2/Git-Tools-Submodules" class="inline-onebox">Git - Submodules</a></p>
